gr-digital/lib/correlate_access_code_ff_ts_impl.cc

```diff
@@ -138,8 +139,12 @@ int correlate_access_code_ff_ts_impl::general_work(int noutput_items,
                 // shift in new data
                 d_data_reg =
                     (d_data_reg << 1) | (gr::branchless_binary_slicer(in[count++]) & 0x1);
-
-                // compute hamming distance between desired access code and current data
+                if (d_data_reg_bits < d_len) {
```

Comment:
Should increment before compare (minor).

Reply:
Done. Also added test.